components/about/about.tsx

```diff
@@ -1,36 +1,88 @@
 import Image from "next/image";
-import MyPicture from "../../assets/MyPicture.png";
-import React from "react";
+import MyPicture from "../../assets/MyProfilePicture.png";
+import React, { useEffect, useState } from "react";
 
 export default function AboutMe() {
+  const [isOnPosition, setIsOnPosition] = useState<boolean>(false);
+
+  const scrollHeader = () => {
+    if (window.scrollY >= 100) {
+      setIsOnPosition(true);
+    }
+  };
+  useEffect(() => {
+    window.addEventListener("scroll", scrollHeader);
+    return () => {
+      window.addEventListener("scroll", scrollHeader);
+    };
+  }, []);
+  const [opacity, setOpacity] = useState<number>(0);
+
+  useEffect(() => {
+    const fadeInDuration = 3000;
```

Comment:
You can take fadeInDuration out of here, and declare on the top as well, due to maintainance.